File: api-examples/cat/count/js/list_All_Items_ItemType.js
var settings = {
    "async": true,
    "crossDomain": true,
    "url": "https://catalogue.iudx.org.in/catalogue/v1/search/catalogue/v1/count",
    "method": "GET",
  }
  
  $.ajax(settings).done(function (response) {
    console.log(response);
  });

File: api-examples/cat/search/python/list_All_Items_ItemType.py
import requests

url = "https://<catalogue-server-ip>/catalogue/v1/search"

payload = {}
headers= {}

response = requests.request("GET", url, headers=headers, data = payload)

print(response.text.encode('utf8'))
import requests

url = "https://catalogue.iudx.org.in/catalogue/v1/search/catalogue/v1/search"

payload = ""

response = requests.request("GET", url, data=payload)

print(response.text)
